x-pack/auditbeat/processors/add_session_metadata/add_session_metadata_test.go

```diff
@@ -0,0 +1,211 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package add_session_metadata
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/elastic/beats/v7/libbeat/beat"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/pkg/processdb"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/pkg/procfs"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
github.com/stretchr in the elastic stanza will make `make check` unhappy.

Reply:
I've formatted all the files, it rearranged all the imports